sdk/servicebus/service-bus/test/atomE2ETests.spec.ts

```diff
@@ -87,3 +87,58 @@ describe("Filter messages with the rules set by the ATOM API", () => {
 
   // TODO: New tests for rule filters to match the sent messages can be added
 });
+
+describe("getSubscriptionRuntimeProperties", () => {
+  const topicName = "new-topic-2";
+  const subscriptionName1 = "new-subscription-1";
+  const subscriptionName2 = "new-subscription-2";
+  const serviceBusClient = new ServiceBusClient(getConnectionString());
+
+  beforeEach(async () => {
+    await recreateTopic(topicName);
+  });
+
+  after(async () => {
+    await serviceBusClient.close();
+    await serviceBusAtomManagementClient.deleteTopic(topicName);
+  });
+
+  it("Active Message Count - single subscription", async () => {
+    await recreateSubscription(topicName, subscriptionName1);
+    const messages = [1, 2, 3].map((num) => {
+      return {
+        body: `msg-${num}`
+      };
+    });
+    await serviceBusClient.createSender(topicName).sendMessages(messages);
+    const activeMessageCount = (
+      await serviceBusAtomManagementClient.getSubscriptionRuntimeProperties(
+        topicName,
+        subscriptionName1
+      )
+    ).activeMessageCount;
+    chai.assert.equal(activeMessageCount, messages.length, "Unexpected active message count");
+  });
+
+  it("Active Message Count - multiple subscriptions", async () => {
+    await recreateSubscription(topicName, subscriptionName1);
+    await recreateSubscription(topicName, subscriptionName2);
+    const messages = [1, 2, 3].map((num) => {
+      return {
+        body: `msg-${num}`
+      };
+    });
+    await serviceBusClient.createSender(topicName).sendMessages(messages);
+
+    for await (const subscription of serviceBusAtomManagementClient.listSubscriptionsRuntimeProperties(
+      topicName
+    )) {
+      chai.assert.equal(
+        subscription.activeMessageCount,
+        messages.length,
+        "Unexpected active message count"
+      );
+    }
+  });
+  // TODO: New E2E tests can be added
```

Comment:
Is there an issue for this?

Reply:
I was thinking we'd only create the tests as needed. Otherwise, we'd be basically testing the service behavior.
Hence, not creating a new issue.

Reply:
Let me know if you think otherwise.

Reply:
In that case...can we remove the TODO?

Reply:
okay, fine!
```suggestion
```